.pdd

```diff
@@ -18,5 +18,5 @@
 --exclude test/test_pdd.rb
 --exclude src/main/resources/images/**/*
 --rule min-words:20
---rule min-estimate:15
+--rule min-estimate:0
```

Comment:
@Doldrums why? I suggest we keep it to 15

Reply:
oh sorry, that was used for testing. I will return now